message_definitions/v1.0/ras_a.xml

```diff
@@ -0,0 +1,124 @@
+      <field type="uint8_t" name="flags" display="bitmask">Bitmask for POI state. Bit 1: POI is in focus on camera, Bit 8: POI has been cleared and should be deleted.</field>
+      <field type="uint64_t" name="uid" invalid="0">Unique ID for a given POI. Updates to a POIs information should use the same uid. 0 means unknown.</field>
+      <field type="int32_t" name="latitude" units="degE7" invalid="INT32_MAX">Latitude (WGS84) of the POI. If unknown: INT32_MAX (both Lat/Lon).</field>
+      <field type="int32_t" name="longitude" units="degE7" invalid="INT32_MAX">Longitude (WGS84) of the POI. If unknown: INT32_MAX (both Lat/Lon).</field>
+      <field type="int32_t" name="alt_amsl" units="mm" invalid="INT32_MAX">Altitude AMSL of the POI. Positive for up. INT32_MAX if unknown.</field>
+      <field type="int32_t" name="alt_ellip" units="mm" invalid="INT32_MAX">Altitude above the EGM96 ellipsoid of the POI. Positive for up. INT32_MAX if unknown.</field>
+      <field type="uint32_t" name="classification" enum="POI_CLASSIFICATION">Classification of the POI. Can either used the POI_CLASSIFICATION enumeration (0x0 - 0x7FFFFFFF reserved), or use the reserved range for implementation specific classifications (0x80000000 - UINT32_MAX).</field>
+      <field type="float" name="x" units="m" invalid="NaN">X position of the POI in the NED frame. NAN if unknown.</field>
+      <field type="float" name="y" units="m" invalid="NaN">Y position of the POI in the NED frame. NAN if unknown.</field>
+      <field type="float" name="z" units="m" invalid="NaN">Z position of the POI in the NED frame. NAN if unknown.</field>
+      <field type="float[4]" name="q" invalid="[NAN]">Orientation quaternion (w, x, y, z order) of the POI in the NED frame. Zero-rotation is 1, 0, 0, 0. Unknown is NAN, NAN, NAN, NAN.</field>
+      <field type="float" name="dist" units="m" invalid="NaN">Distance from the aircraft sensor/camera focal point to the POI. NAN if unknown.</field>
+      <field type="float" name="vel_n" units="m/s" invalid="NaN">North velocity of the POI. NAN if unknown.</field>
+      <field type="float" name="vel_e" units="m/s" invalid="NaN">East velocity of the POI. NAN if unknown.</field>
+      <field type="float" name="vel_d" units="m/s" invalid="NaN">Down velocity of the POI. NAN if unknown.</field>
+      <field type="float" name="hdg" units="rad" invalid="NaN">Heading of the POI in the NED frame. NAN if unknown.</field>
+      <field type="float" name="height" units="m" invalid="NaN">Height of the POI shape. When the geometry is a circle, sphere or cylinder, represents the radius. NAN if unknown.</field>
+      <field type="float" name="width" units="m" invalid="NaN">Width of the POI shape. NAN if unknown.</field>
+      <field type="float" name="depth" units="m" invalid="NaN">Depth of the POI shape. NAN if unknown.</field>
```

Comment:
What is the width and depth of a pyramid?  Or to put it anyway, should we specify "Maximum width" etc.

Reply:
Doesn't make much sense to consider a pyramid. That's why it's out of the enumeration.